react-redux/.github/workflows/linters.yml

```diff
@@ -16,7 +16,7 @@ jobs:
           node-version: "12.x"
       - name: Setup ESLint
         run: |
-          npm install --save-dev eslint@6.8.x eslint-config-airbnb@18.1.x eslint-plugin-import@2.20.x eslint-plugin-jsx-a11y@6.2.x eslint-plugin-react@7.20.x eslint-plugin-react-hooks@2.5.x babel-eslint@10.1.x
+          npm install --save-dev eslint@7.11.x eslint-config-airbnb@18.1.x eslint-plugin-import@2.20.x eslint-plugin-jsx-a11y@6.2.x eslint-plugin-react@7.20.x eslint-plugin-react-hooks@2.5.x babel-eslint@10.1.x
```

Comment:
@Spayco Is this version tested? Can you use it in a test repo and see if it work as expected before we merge it? Thanks!

Reply:
hey @bolah2009, yes, i tested it, i'm currently working on a react project using it, and it's working fine with 0 issues

here's the repo: https://github.com/Spayco/gtime-frontend/tree/feature (not complete, but should be functional)

Reply:
@Spayco I meant on GHA. Does the workflow config still work on Github Actions without any issue?

Reply:
@bolah2009 it would work fine, but right now, there's an error with `eslint-config-airbnb-base@14.2.1` that's affecting everyone on pull requests (even those who didn't update ESLint)
![image](https://user-images.githubusercontent.com/11758151/98950832-c970c680-24f9-11eb-9123-79aac1724e85.png)

so we would have to downgrade it to 14.2.0 to fix it. 
I will add that as well, and from there no other issue would pop up

this is a PR from the same project: https://github.com/Spayco/gtime-frontend/pull/4/checks?check_run_id=1390814145

[19844d3](https://github.com/Spayco/gtime-frontend/pull/4/commits/19844d34d4da6b65e54f9c7f690f806498e348bf) is the one with the old ESLint
[1cdeb80](https://github.com/Spayco/gtime-frontend/pull/4/commits/1cdeb803ebff551e0d2d863b153e8b4fdc61601e) is the one with the new ESLint

they both work without issues.